devices/CharacterLcd/custom/SpanPinValuePair.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System.Device.Gpio;
```

Comment:
missing header as well

Reply:
I have added a header (and a comment that the autogenerator (where the class was copied from should also handle this)